xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecTX3G.cpp

```diff
@@ -127,6 +127,8 @@ OverlayMessage CDVDOverlayCodecTX3G::Decode(DemuxPacket* pPacket)
     text = data + sampleData.GetPosition();
     sampleData.SkipChars(textLength);
   }
+  if (!text)
```

Comment:
Unsure this is the correct message to return, this means the decoder needs more data. Should it return OC_error instead?

Reply:
maybe yes done